docs/index.md

```diff
@@ -0,0 +1,58 @@
+---
+hide: toc
+---
+
+# Documentation for pynxtools-mpes
+
+pynxtools-mpes is a free, and open-source data software for harmonizing multidimensional photoemission spectroscopy (MPES) data and metadata for research data management using [NeXus](https://www.nexusformat.org/).
+
+pynxtools-mpes is a plugin for [pynxtools](https://github.com/FAIRmat-NFDI/pynxtools) for translating MPES and standardizing such that it is compliant with the NeXus application definition [`NXmpes`](https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXmpes.html). In addition, it also supports the specialized application definition [`NXmpes_arpes`](https://fairmat-nfdi.github.io/nexus_definitions/classes/contributed_definitions/NXmpes_arpes.html) for angle-resolved photoemission spectroscopy (ARPES) experiments.
+
+pynxtools-mpes is developed both as a standalone reader and as a tool within [NOMAD](https://nomad-lab.eu/), which is the open-source data management platform for materials science we are developing within [FAIRmat](https://www.fairmat-nfdi.eu/fairmat/).
+
+<div markdown="block" class="home-grid">
+<div markdown="block"> 
+
+### Tutorial
+
+A series of tutorials giving you an overview on how to store or convert your XPS data to NeXus compliant files.
+
+- [Installation guide](tutorial/installation.md)
+- [Standalone usage](tutorial/standalone.md)
+
+</div>
+<div markdown="block">
+
+### How-to guides
+
+How-to guides provide step-by-step instructions for a wide range of tasks, with the overarching topics:
+
+- [How to build a new MPES reader](how-tos/build-a-reader.md)
+
+</div>
+
+<div markdown="block">
+
+### Learn
+
+- [Examples](explanation/examples.md)
```

Comment:
Not sure the examples fit within "learn". Isn't this more for providing background information? I think the examples are more how-to guides.

Reply:
moved it together with the standalone.md file inside tutorials